Core/src/main/java/com/plotsquared/core/PlotSquared.java

```diff
@@ -1574,6 +1580,13 @@ public void registerCaptionMap(
         return this.plotListener;
     }
 
+    /**
+     * Get if the {@link PlatformReadyEvent} has been sent by WE. There is no way to query this within WE itself.
```

Comment:
```suggestion
     * Get if the {@link PlatformReadyEvent} has been sent by WorldEdit. There is no way to query this within WorldEdit itself.
     * @since TODO
```